### PR TITLE
[INFRA-2346]  Build & Deploy for WebSDK

### PR DESCRIPTION
**Pull request by JaveriaK**
@rubinsingh @yntema @hubt 

ToDo:
- enable logic for committing changes back to repo after creating a public access key

### CODE REVIEW

File: deployment/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+#chmod 600 ~/deploy/Key.pem
+#git config --global user.email "buildbot@branch.io" && git config --global user.name "Build Bot"
+#git config --global push.default simple
+
+#echo "Commiting changes back to repo"
+#git add dist/build.min.js.gz
+#git commit -am "Tagging release $VERSION [ci skip]"
+#git add $HOME/$CIRCLE_PROJECT_REPONAME/CHANGELOG.md
+#git tag $VERSION
```
Comment (yntema) [inline deployment/deploy.sh]: Could you add a comment about why this section is commented out?

File: deployment/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+make clean
+make
+
+# Cleaning up backup files
+echo "Cleaning up backup files ..."
+rm -f bower.json-e CHANGELOG.md-e package.json-e src/0_config.js-e test/web-config.js-e
+
+#echo "Pushing changes back to repo ..."
+#git push 
+#git push origin $VERSION
```
Comment (yntema) [inline deployment/deploy.sh]: Do we need to manually add the tagged version to github?

Comment (JaveriaK) [inline deployment/deploy.sh]: For now yes, when this section starts working it will push the tag for you.

File: deployment/deploy.sh (hunk 1 above)
Comment (JaveriaK) [inline deployment/deploy.sh]: done

File: circle.yml [hunk 3]
```diff
@@ -23,8 +24,14 @@ dependencies:
     - which grunt 2>&1 >/dev/null || npm install -g grunt-cli
   cache_directories:
     - "~/nvm/v0.10.31"
```
Comment (rubinsingh) [inline circle.yml]: I think this should be "~/nvm/6.11.2"

**Pull request merged**